src/components/Button/FolderButton/FolderButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { MouseEvent, ReactNode } from "react";
+import styles from "./folderButton.module.css";
+
+interface Props {
+  children: ReactNode;
+  onClick: (e: MouseEvent<HTMLButtonElement>) => void;
+  ischecked: boolean;
+}
```

Comment:
사소하지만 일반적인 값이랑 함수는 구분지어서 나열해주시면 코드를 파악하기에 더 편리할 것 같아요~
```suggestion
interface Props {
  ischecked: boolean;
  children: ReactNode;
  onClick: (e: MouseEvent<HTMLButtonElement>) => void;
}
```